### PR TITLE
[IT] Add `Medals`

### PR DESCRIPTION
**Pull request by arancinosbarazzino**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Medals/it.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Tutte le Medaglie di Introduzione Alle Mod possono essere sbloccate in qualunque [modalità di gioco](/wiki/Game_mode) a meno che non venga specificato diversamente.
+
+## Hush-Hush
+
+Le **medaglie Hush-Hush** sono medaglie "segrete" ottenute compiendo azioni molto specifiche, che non vengono esplicitamente dette al giocatore (da qui proviene il nome). Queste azioni possono variare da completare una beatmap in particolari circostanze a stabilire una giocata con una combinazione di mod inusuale. Nonostante la loro natura segreta, ci sono degli indizi nascosti in giro per aiutare tutti i collezionistsi di medaglie curiosi nella loro ricerca come segue: 
+
+- Il nome della medaglia in sé,
+- Le brevi descrizioni delle medaglie,
+- I loro *testi a comparsa*<!-- https://twitter.com/ephemeralis/status/1068054191602638850 --> delle medaglie (se ci sono), visibile passando sopra l'emblema con il cursore.
```
Comment (Sclafus) [inline wiki/Medals/it.md:35]:
```suggestion
- I loro *testi a comparsa*<!-- https://twitter.com/ephemeralis/status/1068054191602638850 --> (se ci sono), visibili passando sopra l'emblema con il cursore.
```

File: wiki/Medals/it.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+Siccome le medaglie Hush-Hush sono fatte per essere obiettivi segreti, i requisiti per sbloccarle non sono vengono direttamente detti al pubblico; ad ogni modo, i giocatori possono [collaborare per capire insieme un modo per ottenere tutte le medaglie Hush-Hush qui](https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals).
```
Comment (Sclafus) [inline wiki/Medals/it.md:37]:
```suggestion
Siccome le medaglie Hush-Hush sono fatte per essere obiettivi segreti, i requisiti per sbloccarle non sono vengono direttamente rivelati al pubblico; ad ogni modo, i giocatori possono [collaborare per capire insieme un modo per ottenere tutte le medaglie Hush-Hush qui](https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals).
```

File: wiki/Medals/it.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+- [Mods di riduzione della difficoltà](/wiki/Game_modifier#difficulty-reduction)
+- [Mods di aumento della difficoltà](/wiki/Game_modifier#difficulty-increase)
+- [xK](/wiki/Game_modifier/xK), [Mirror](/wiki/Game_modifier/Mirror), e [Spun Out](/wiki/Game_modifier/Spun_Out)
+
+Nonostante abbiamo smesso di creare Beatmap Spotlights Mensili dall'aprile del 2018 per dare posto agli [Spotlights Stagionali](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018), tutte le medaglie pertinenti ad esso possono essere ottenute ancora oggi.
+
+## Seasonal Spotlights
+
+le **medaglie Spotlights Stagionali** sono medaglie ottenute completando determinati pacchetti beatmap rilasciati come parte degli [Spotlights Stagionali](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018). Per sbloccare le medaglie in questa categoria, un giocatore deve giocare tutte le mappe *in qualunque pacchetto delle 4 modalità di gioco* che fanno parte del corrispondente schema di classificazione degli Spotlights Stagionali. È possibile utilizzare le seguenti mods:
```
Comment (Sclafus) [inline wiki/Medals/it.md:51]:
```suggestion
Le **medaglie Spotlights Stagionali** sono medaglie ottenute completando determinati pacchetti beatmap rilasciati come parte degli [Spotlights Stagionali](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018). Per sbloccare le medaglie in questa categoria, un giocatore deve giocare tutte le mappe *in qualunque pacchetto delle 4 modalità di gioco* che fanno parte del corrispondente schema di classificazione degli Spotlights Stagionali. È possibile utilizzare le seguenti mods:
```

File: wiki/Medals/it.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+Nonostante abbiamo smesso di creare gli Spotlights Stagionali dall'inizio 2020 in favore dell'[attuale sistema di Beatmap Spotlights](/wiki/Beatmap_Spotlights), tutte le medaglie pertinenti ad esso possono essere ottenute ancora oggi.
+
+## Pacchetti Sfida Beatmap
+
+le **medaglie dei Pacchetti Sfida Beatmap**, o alternativamente conosciute come [*medaglie Pacchetti Abilità*](https://osu.ppy.sh/home/news/2020-11-20-featured-artist-beatmap-updates-from-the-mappers-guild#skill-packs), sono medaglie ottenute al completamento della difficoltà più alta presente in tutte le beatmap presenti in alcuni [Pacchetti Beatmap a Tema](https://osu.ppy.sh/beatmaps/packs?type=theme) o [Pacchetti Beatmap Artista/Album](https://osu.ppy.sh/beatmaps/packs?type=artist) *senza alcuna mod che [riducono la difficoltà](/wiki/Game_modifier#difficulty-reduction) o [speciali](/wiki/Game_modifier#special) abilitate*. Questi pacchetti beatmap sono curati appositamente per mettere alla prova le abilità di ogni giocatore in vari aspetti del gioco riguardanti diversi livelli di abilità, ad esempio all'interno di una particolare gamma di [difficoltà](/wiki/Beatmapping/Star_rating) o con una difficoltà massima.
```
Comment (Sclafus) [inline wiki/Medals/it.md:61]:
```suggestion
Le **medaglie dei Pacchetti Sfida Beatmap**, o altrimenti conosciute come [*medaglie Pacchetti Abilità*](https://osu.ppy.sh/home/news/2020-11-20-featured-artist-beatmap-updates-from-the-mappers-guild#skill-packs), sono medaglie ottenute al completamento della difficoltà più alta presente in tutte le beatmap presenti in alcuni [Pacchetti Beatmap a Tema](https://osu.ppy.sh/beatmaps/packs?type=theme) o [Pacchetti Beatmap Artista/Album](https://osu.ppy.sh/beatmaps/packs?type=artist) *senza alcuna mod che [riduca la difficoltà](/wiki/Game_modifier#difficulty-reduction) o [speciali](/wiki/Game_modifier#special) abilitate*. Questi pacchetti beatmap sono curati appositamente per mettere alla prova le abilità di ogni giocatore in vari aspetti del gioco riguardanti diversi livelli di abilità, ad esempio all'interno di una particolare gamma di [difficoltà](/wiki/Beatmapping/Star_rating) o con una difficoltà massima.
```

File: wiki/Medals/it.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+tutte le beatmap nel pacchetto devono essere completate nella modalità di gioco [osu!](/wiki/Game_mode/osu!) per poter ottenere la medaglia, a meno che diversamente specificato.
```
Comment (Sclafus) [inline wiki/Medals/it.md:63]:
```suggestion
Tutte le beatmap nel pacchetto devono essere completate nella modalità di gioco [osu!](/wiki/Game_mode/osu!) per poter ottenere la medaglia, a meno che diversamente specificato.
```

**Review by Sclafus — CHANGES_REQUESTED** [review]

**Review by Sclafus — APPROVED** [review]

**Comment by TicClick** [comment]
please include changes from https://github.com/ppy/osu-wiki/pull/8735 as well (it's a small one)

**Review by Sclafus — APPROVED** [review]
The newly added part is also good

**Review by TicClick — APPROVED** [review]

**Pull request merged**